numpy/random/src/mt19937/mt19937-poly.h

```diff
@@ -1,3 +1,11 @@
+/*
+ * 2**128 step polynomial produced using the file mt19937-generate-jump-poly.c
+ * (randomgen) which is a modified version of minipoly_mt19937.c as distributed
+ * in http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/JUMP/jump_ahead_1.02.tar.gz
+ *
+ * These files are not part of NumPy.
```

Comment:
Makes sense and is easily readable! Do you think it makes sense to refer to the commit (5055c43) or a maybe rather version number (1.18.1 ?) of randomgen, too?